src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -589,20 +588,47 @@ def _generate_tool_meta(
     timeout: int,
     *,
     include_errors_in_output: bool = False,
+    load_in_subprocess: bool = True,
```

Comment:
Do we have a test for case where `load_in_subprocess` is set to false?

Reply:
yes we have. there are tests for `_generate_tools_meta`, where `load_in_subprocess` will be set to `False`